### ISSUE TITLE
Use Market Time

### ISSUE THREAD

**Issue by gabriellet** [gabriellet/thepriceisright#18]
Currently we use local machine time. The server returns a time. We should use this instead.

**Comment by gabriellet** [comment]
[Python date time strptime](https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior)

**Comment by ls3233** [comment]
API Examples
============
See also [client.py](https://github.com/texodus/exchange_simulator/blob/master/client.py)

Query:

	$ curl 'http://localhost:8080/query?id=1'
	{"id": "1", "top_ask": {"price": 129.18, "size": 70}, "timestamp": "2016-08-06 12:32:11.821574", "top_bid": {"price": 128.79, "size": 61}}

Filled order:

	$ curl 'http://localhost:8080/order?id=2&side=sell&qty=100&price=125.0'
	{"id": "2", timestamp": "2016-08-06 12:32:20.860511", "qty": 100.0, "side": "sell", "avg_price": 129.5}

Unfilled order:

	$ curl 'http://localhost:8080/order?id=3&side=sell&qty=100&price=135.0'
	{"id": "3", timestamp": "2016-08-06 12:32:24.990886", "qty": 0, "side": "sell", "avg_price": 0}

**Comment by gabriellet** [comment]
Format string: `'%Y-%m-%d %H:%M:%S.%f'`
Usage:
```
d = "2016-08-06 12:32:24.990886" //some timestamp
f = '%Y-%m-%d %H:%M:%S.%f'
dt = datetime.datetime.strptime(d, f)
//dt is now datetime object: datetime.datetime(2016, 8, 6, 12, 32, 24, 990886)
```

**Comment by gabriellet** [comment]
Please name variable `f` as `FORMAT` wherever used

**Comment by gabriellet** [comment]
We need to see if the `query_market_datetime()` function really needs to check if selling failed since the first response it gets will have a timestamp

**Comment by gabriellet** [comment]
Fixed 209aee2
